### PR TITLE
refactor: remove format_xxx settings.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes https://github.com/datafuselabs/databend/issues/8995

some futher refactor need to be done, after this pr is merged.

### CODE REVIEW

**Review by Xuanwo — APPROVED** [review]

**Comment by BohuTANG** [comment]
We should document them as deprecated and instead use the new formats.

**Comment by youngsofun** [comment]
> We should document them as deprecated and instead use the new formats.

docs are already updated to the new format last week.

**Comment by youngsofun** [comment]
1. old format is not mentioned in current docs
2. when using old format  will get error message like "Only supports INSERT INTO $table FILE_FORMAT = (type = 'CSV' ..)"

File: src/query/service/src/servers/http/v1/load.rs [hunk 1]
```diff
@@ -229,8 +178,16 @@ pub async fn streaming_load(
                     )),
                 }
             }
+            InsertInputSource::StreamingWithFormat(_, _, _) => Err(poem::Error::from_string(
+                "'INSERT INTO $table FORMAT <type> is now only supported in clickhouse handler,\
+                     please use 'FILE_FORMAT = (type = <type> ...)' instead.",
+                StatusCode::BAD_REQUEST,
+            )),
```
Comment (youngsofun) [inline src/query/service/src/servers/http/v1/load.rs:185]: @BohuTANG refined the error message.

**Review by ariesdevil — APPROVED** [review]

**Pull request merged**